packages/@ourworldindata/grapher/src/controls/Controls.scss

```diff
@@ -1,171 +1,483 @@
-    margin-right: 2px;
-}
+    .map-projection-menu {
+        font: $medium 13px/16px $lato;
+
+        .control {
+            min-height: auto;
+            min-width: 150px;
+            font: $medium 13px/16px $lato;
+            letter-spacing: 0.01em;
+            display: flex;
+            align-items: center;
+            border: 1px solid $light-stroke;
+            border-radius: 4px;
+            padding: 7px;
+            color: $dark-text;
+
+            &:hover {
+                background: $hover-fill;
+                cursor: pointer;
+            }
 
-.selected .FacetStrategyPreview-none-child {
-    background: #d9eafb;
-}
+            &:after {
+                content: " ";
+                background: url($menu-caret-down) no-repeat center;
+                width: 16px;
+                height: 16px;
+            }
 
-.FacetStrategyPreview-none-child {
-    background: #dde1e7;
-}
+            &.active {
+                border-color: $active-stroke;
+                &:after {
+                    background: url($menu-caret-up) no-repeat center;
+                }
+            }
+        }
 
-.selected .FacetStrategyPreview-split-child {
-    background: #d9eafb;
-}
+        .menu {
+            margin-top: 3px;
+            border-radius: 4px;
+            background: white;
+            box-shadow: 0px 4px 40px 0px rgba(0, 0, 0, 0.15);
+            z-index: $zindex-controls-drawer;
+
+            &:before {
+                display: inline-block;
+                content: "Zoom to selection";
+                font: 700 12px/16px $lato;
+                color: $light-text;
+                padding: 8px 18px;
+            }
 
-.FacetStrategyPreview-split-child {
-    background: #dde1e7;
+            .option {
+                padding: 8px 18px;
+                &:hover {
+                    cursor: pointer;
+                    background: $hover-fill;
+                }
+                &:active,
+                &.active {
+                    color: $active-text;
+                    background: $active-fill;
+                }
+                &.active {
+                    position: relative;
+                    &:hover {
+                        background: $selected-fill;
+                    }
+                    &:after {
+                        content: " ";
+                        background: url($option-checkmark) no-repeat;
+                        width: 12px;
+                        height: 9px;
+                        position: absolute;
+                        right: 18px;
+                        bottom: 11px;
+                    }
+                }
+            }
+        }
+    }
 }
 
-@keyframes bounceIn {
-    from,
-    20%,
-    40%,
-    60%,
-    80%,
-    to {
-        animation-timing-function: cubic-bezier(0.215, 0.61, 0.355, 1);
+@at-root {
+    // collapse the Settings toggle down to just an icon on small screens
+    .GrapherComponentSmall:not(.GrapherComponentMedium)
```

Comment:
I don't think `:not(.GrapherComponentMedium)` is needed here?

Aside, you could also do `&.GrapherComponentSmall` instead of `@at-root` if you wanted to since `Controls.scss` is imported within `.GrapherComponent {}`

Reply:
Ah I can see how these `GrapherComponentMedium/Small/Narrow` classes are annoying in CSS if they're not mutually exclusive. If you want to you could update the [css classes](https://github.com/owid/owid-grapher/blob/e2073c69ffeb37a19aa659540c4721519fff4827/packages/%40ourworldindata/grapher/src/core/Grapher.tsx#L2297-L2299) to be mutually exclusive. I haven't used these a lot, so it should not be much work to update the few instances where I do. It would be important though not to touch `grapher.isNarrow`, `grapher.isSmall`, etc. themselves, that would mess things up in code..

Reply:
Ultimately I think we'll need to do the label-hiding in the entity-selector and settings buttons in js since it will depend on the number of content-switcher buttons, the text-width of the entity name and the possible presence of a 'promoted' single control (replacing the settings button when it's the only option present). 

Until then, I'm using CSS container queries with a pair of breakpoints chosen for the common (worst) case of:
- a three-segment content swticher
- "countries or regions" as the entity name 
- a visible Settings menu